### DISCUSSION TITLE
查看账户余额

### DISCUSSION THREAD

**Discussion by wuqinqiang** [BurnHair/docs#190] [分享讨论]
我只在令牌页面上看到各令牌消耗情况，哪里可以看自己账户的余额。

**Comment by BennyThink** [comment]
在这个页面 https://burn.hair/token

拉到下面，有一个统计
<img width="615" alt="image" src="https://github.com/user-attachments/assets/a817b354-c484-4972-8b5e-44a2512bbba3">

会写明余额等信息
